packages/backend/src/main.ts

```diff
@@ -9,7 +9,11 @@ import { BACKEND_PORT } from './util/environment'
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule)
-
+  app.enableCors({
+    origin: ['http://localhost:3000'],
+    methods: ['GET', 'POST'],
+    credentials: true,
```

Comment:
The url should be in .env. And we have patch and delete requests, maybe even put